eve/workers/end2end/scripts/run-e2e-test.sh

```diff
@@ -16,7 +16,16 @@ BACKBEAT_BUCKET_CHECK_TIMEOUT_S=${BACKBEAT_BUCKET_CHECK_TIMEOUT_S:-10}
 POD_NAME="${ZENKO_NAME}-${STAGE}-test"
 TOKEN=$(get_token)
 
+CLOUDSERVER_SECRET_V4=$(kubectl get secret end2end-connector-cloudserver-config.v4 -o jsonpath="{.data.config\.json}}" | base64 -di)
```

Comment:
In addition, the `v4` may be flacky (or need mainteance) : depending on the scenario/state of cluster, the config may not be in end2end-connector-cloudserver-config.v4, but in another versoin of the object...

to get the object, we may list object by label : `kubectl get secret -l app.kubernetes.io/name=connector-cloudserver-config,app.kubernetes.io/instance=end2end`

Somehow, this however changes the jsonpath expression : since we get a _list_ of resources (even if there is only one element):

```suggestion
CLOUDSERVER_SECRET_V4=$(kubectl get secret -l app.kubernetes.io/name=connector-cloudserver-config,app.kubernetes.io/instance=end2end -o jsonpath="{.items[0].data.config\.json}}" | base64 -di)
```